web3/contract.py

```diff
@@ -128,12 +129,13 @@
     WARN,
     EventLogErrorFlags,
 )
-from web3.types import (
+from web3.types import (  # noqa: F401
```

Comment:
this `noqa` looks suspect (like it probably isn't needed...)

Reply:
It's needed for the `LogParams` type hint on line `1087` - I'm not sure why flake isn't picking up on it